src/Stormpath/Resource/CustomData.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Stormpath\Resource;
+
+
+use Stormpath\Stormpath;
+
+class CustomData extends AbstractCollectionResource implements Saveable {
+    const PATH = 'customData';
+
+    public function __get($key)
+    {
+       return $this->getProperty($key);
+    }
+
+    public function __set($key, $value)
+    {
+        $this->setProperty($key, $value);
+    }
+
+    public function getItemClassName()
+    {
+        return Stormpath::CUSTOM_DATA;
+    }
+
+    public function save()
+    {
+        return $this->getDataStore()->save($this);
```

Comment:
Take a look at the Java SDK implementation to see if this method needs some more work: https://github.com/stormpath/stormpath-sdk-java/blob/master/impl/src/main/java/com/stormpath/sdk/impl/resource/AbstractExtendableInstanceResource.java